sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -50,57 +51,110 @@
         this.service = service;
     }
 
-    Mono<Response<DocumentResultCollection<AnalyzeSentimentResult>>> analyzeSentimentBatchWithResponse(
+    /**
+     * Helper function that analyzes a batch of documents and returns {@link TextAnalyticsPagedFlux} that is a paged
+     * flux contains {@link AnalyzeSentimentResult}.
+     *
+     * @param textInputs A batch of input texts.
```

Comment:
Need to make sure that the naming is constant for this one. Either documents, inputs or textInputs etc.